gettext_i18n_rails.gemspec

```diff
@@ -15,7 +15,8 @@ Gem::Specification.new name, GettextI18nRails::VERSION do |s|
   s.add_development_dependency "haml"
   s.add_development_dependency "rake"
   s.add_development_dependency "rails"
-  s.add_development_dependency "ruby_parser", ">= 3"
+  s.add_development_dependency "ruby_parser", ">= 3.7.1"
```

Comment:
yes that's what I meant :)

should be fine to force old users to upgrade ruby_parser ...

On Wed, Oct 7, 2015 at 9:33 AM, Martin Povolny <notifications@github.com>
wrote:

> In gettext_i18n_rails.gemspec
> <https://github.com/grosser/gettext_i18n_rails/pull/142#discussion_r41413121>
> :
>
> > @@ -15,7 +15,8 @@ Gem::Specification.new name, GettextI18nRails::VERSION do |s|
> >    s.add_development_dependency "haml"
> >    s.add_development_dependency "rake"
> >    s.add_development_dependency "rails"
> > -  s.add_development_dependency "ruby_parser", ">= 3"
> > +  s.add_development_dependency "ruby_parser", ">= 3.7.1"
>
> I don't quite understand what you mean? Shall I add the 'gem' call to the
> file that needs the parser? Meaning into
> lib/gettext_i18n_rails/ruby_gettext_extractor.rb?
>
> The version is needed to support ruby 2.2. Older ruby works with older
> versions of the gem.
>
> Please, try to reformulate what you want me to do. I'll do whatever is
> needed for sure ;-)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grosser/gettext_i18n_rails/pull/142/files#r41413121>.
>